.github/PULL_REQUEST_TEMPLATE_nolabels

```diff
@@ -0,0 +1,13 @@
+## ATTENTION - Release Note ack required!
+
+Update the below named `release-note-needed` block to one of:
+
+**\`\`\`release-note**
+**\`\`\`release-note-action-required**
+**\`\`\`release-note-none**
+
+Then describe your change as a summarized release note within the block.
+
+```release-note-needed
+...release-note block...
```

Comment:
Right, this can get a bit complicated if we're not careful and we don't want complicated.  Today, you add a label and the title is the release note. Done.

With this new template we need to figure out how to make it simple to use but get what we want.  Since you've stated that we absolutely want to keep labels we can work with that.  Something like:

* labels determine state and category as they do today
* We can pre-fill the template block with choices so people don't have to hunt for usage/help.
* template block can be cleared as a way to point to the title for the release note or contain actual content

Maybe something like:

```
Use the release-note-* labels to set the release note state and then clear this block to use the PR title as the release note or enter your extended release note here.
```